src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -311,14 +358,14 @@ private void WorkerLoop()
 
                         // We have one or more active operations. Run any work that needs to be run.
                         CURLMcode performResult;
-                        while ((performResult = Interop.Http.MultiPerform(multiHandle)) == CURLMcode.CURLM_CALL_MULTI_PERFORM);
+                        while ((performResult = Interop.Http.MultiPerform(_multiHandle)) == CURLMcode.CURLM_CALL_MULTI_PERFORM);
```

Comment:
I don't mind adding asserts, but I'm curious about the need here.  If it's null, it'll throw an exception.  If it's invalid, we'll throw an exception in the next line.  And in either case, we actually use the multi handle much earlier in the loop.  Are you suggesting we add such an assert before the loop, on each iteration of the loop, or something else?